### PR TITLE
l10n: Added plural form to string

### PR DESCRIPTION
**Pull request by rakekniven**
Fix for #19304 

Devs: Please verify. Changes are untested due to missing build environment.

Reported at Transifex.

Signed-off-by: rakekniven <mark.ziegler@rakekniven.de>

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

File: apps/settings/src/components/AppList.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<div id="apps-list" class="apps-list" :class="{installed: (useBundleView || useListView), store: useAppStoreView}">
 			<template v-if="useListView">
 				<div v-if="showUpdateAll" class="counter">
-					{{ t('settings', '{counter} apps have an update available', {counter}) }}
+					{{ n('settings', '{counter} app has an update available', '{counter} apps have an update available', {counter}) }}
```
Comment (gary-kim) [inline apps/settings/src/components/AppList.vue]:
```suggestion
					{{ n('settings', '%n app has an update available', '%n apps have an update available', counter) }}
```

**Review by gary-kim — DISMISSED** [review]
I'm not entirely sure this is necessary because this text will only be shown for values > 1 but can't hurt to have in case we change it later.

https://github.com/nextcloud/server/blob/0bb92c027f352118c80f89923a32638955f46724/apps/settings/src/components/AppList.vue#L119-L121

Vue also does not have `n` available in this context at the moment.

You can add `Vue.prototype.n = n` here to fix that:
https://github.com/nextcloud/server/blob/ec01e0a790448fff38364f629a4de4edb5d465bf/apps/settings/src/main-apps-users-management.js#L46-L48

If you'd like, I can push the changes to your branch.

File: apps/settings/src/components/AppList.vue (hunk 1 above)
Comment (rakekniven) [reply apps/settings/src/components/AppList.vue]: @gary-kim Missed your message.

> If you'd like, I can push the changes to your branch.

Yes, please push the changes. I am not that familar with git.

**Review by ChristophWurst — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
please rebase :)

**Comment by gary-kim** [comment]
> please rebase :)

Done!

**Pull request merged**